apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -71,6 +71,19 @@
    */
   "nodeSupportedVersionRange": ">=10.13.0 <11.0.0",
```

Comment:
I tweaked the text for this option also.  I'm trying to replace `NodeJS` in our English text with `Node.js` because that's their official name.